app/models/professional_experience.rb

```diff
@@ -20,13 +20,15 @@ def self.ordered_by_start_date
   def greater_than_current_month?
     return if start_date.to_date < Date.today.end_of_month
 
-    errors.add(:start_date, I18n.t('activerecord.errors.models.professional_experience.attributes.start_date.greater_than_current_month'))
+    errors.add(:start_date,
+               I18n.t('activerecord.errors.models.professional_experience.attributes.start_date.greater_than_current_month'))
```

Comment:
you can remove this whole translation path and let the rails error class to resolve this

https://guides.rubyonrails.org/i18n.html#error-message-scopes

Reply:
you can pass the error type as argument https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add